### DISCUSSION TITLE
请问大家第四课开头安装包的代码报错翻以前的讨论没解决

### DISCUSSION THREAD

**Discussion by L220220** [mlc-ai/mlc-zh#195] [Q&A]
报错ERROR: Could not find a version that satisfies the requirement mlc-ai-nightly (from versions: none)
ERROR: No matching distribution found for mlc-ai-nightly，我看到有个讨论说的汇总的问题，要升级python版本到3.7及以上，可是我的已经是3.8了的

**Comment by ykzhang-eeis** [comment]
#### 说一下我的方法
环境（WSL ubuntu22.04 python 3.10.12）
1. 根据此链接下载whl文件 
https://github.com/mlc-ai/package/releases/download/v0.9.dev0/mlc_ai_nightly-0.12.dev1900-cp310-cp310-manylinux_2_28_x86_64.whl
2. 根据文件所在地址 pip install path/to/mlc_ai_nightly-0.12.dev1900-cp310-cp310-manylinux_2_28_x86_64.whl
